gramps_webapi/api/resources/filters.py

```diff
@@ -0,0 +1,298 @@
+"""Gramps filter interface."""
+
+import json
+from typing import Dict, List, Set
+
+import gramps.gen.filters as filters
+from flask import Response, abort
+from gramps.gen.db.base import DbReadBase
+from gramps.gen.filters import GenericFilter
+from marshmallow import Schema, ValidationError, fields
```

Comment:
Have you forgotten adding `marshmallow` to `setup.py`? I don't know if there are any tests importing this module yet...

Reply:
It is required for webargs, do you think it needs to be added separately?